appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -31,7 +31,10 @@
 from oauth2client.client import GoogleCredentials
 
 
-_FIREBASE_CONFIG = '_firebase_config.html'
+import rest_api
```

Comment:
If you're breaking this into a separate module, then a bunch of the code below needs to be updated to use it.